src/oauth2/client.cr

```diff
@@ -155,16 +163,14 @@ class OAuth2::Client
     get_access_token do |form|
       form.add("grant_type", "refresh_token")
       form.add("refresh_token", refresh_token)
-      form.add "scope", scope unless scope.nil?
+      form.add("scope", scope) unless scope.nil?
     end
   end
 
-  private def get_access_token : AccessToken
-    headers = HTTP::Headers{
-      "Accept"       => "application/json",
-      "Content-Type" => "application/x-www-form-urlencoded",
-    }
-
+  # Makes a token exchange request with custom headers and form fields
+  # Returns a HTTP::Client::Response
```

Comment:
This is superfluous. The return type is sufficiently indicated in the method signature.

```suggestion
```